### PR TITLE
Update communicate-with-users-from-other-organizations.md

### PR DESCRIPTION
**Pull request by kristofftan**
I think for the below sentence, the 'teams' should replace with 'team' may be more accurate. 

"These people cannot be added to the team unless they are invited as guests."

### CODE REVIEW

File: Teams/communicate-with-users-from-other-organizations.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ description: Learn how to call, chat, find, and add users from outside the organ
 
 When you need to communicate and collaborate with people outside your organization, Microsoft Teams has two options:
 
-- **External access** - A type of federation that allows users to find, call, and chat with people in other organizations. These people cannot be added to teams unless they are invited as guests.
+- **External access** - A type of federation that allows users to find, call, and chat with people in other organizations. These people cannot be added to team unless they are invited as guests.
```
Comment (JohanFreelancer9) [inline Teams/communicate-with-users-from-other-organizations.md:27]:
```suggestion
- **External access** - A type of federation that allows users to find, call, and chat with people in other organizations. These people cannot be added to a team unless they are invited as guests.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kristofftan, kindly commit the minor suggested change. Many thanks.

**Comment by kristofftan** [comment]
got it.

**Review by kristofftan — COMMENTED** [review]
Reviewed.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**